packages/webapp/pages/sources/index.tsx

```diff
@@ -102,7 +103,7 @@ const SourcesPage = ({
   return (
     <>
       <NextSeo {...seo} />
-      <main className="py-6 tablet:px-4 laptop:px-10">
+      <PageWrapperLayout className="py-6">
```

Comment:
Should this still have the py-6 then?

Reply:
Due to internal layout we need the padding here for mobile because the button end really near to the header 
<img width="692" alt="image" src="https://github.com/user-attachments/assets/f65fcb63-7628-4639-a43a-fee13fba531b">

Reply:
Ah i see, mainly wanted to avoid it having some impact on breadcrumbs (same as below basically)